pyvista/core/filters/data_set.py

```diff
@@ -4361,9 +4371,22 @@ def streamlines_from_source(
         elif vectors is None:
             pyvista.set_default_active_vectors(self)
 
-        if max_time is None:
-            max_velocity = self.get_data_range()[-1]
```

Comment:
BTW, this was also a bug in that this gets the maximum of the active scalars, NOT the maximum of the magnitude of the active vectors.